ObjectHashServerr.API/Function1.cs

```diff
@@ -0,0 +1,109 @@
+using System;
+using System.IO;
+using System.Net;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Attributes;
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Enums;
+using Microsoft.Extensions.Logging;
+using Microsoft.OpenApi.Models;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using ObjectHashServer.BLL.Models;
+using ObjectHashServer.BLL.Models.Api.Request;
+using ObjectHashServer.BLL.Models.Api.Response;
+using ObjectHashServer.BLL.Services.Implementations;
+using ObjectHashServer.Controllers;
+
+namespace ObjectHashServerr.API
+{
+    public class Function1
```

Comment:
Guess Function1 will be replaced with a meaningfull name

Reply:
@tbordi-ex  rename the class and file to ObjectHasherFunctions